### DISCUSSION TITLE
Server/Client/Both preferred active scene option

### DISCUSSION THREAD

**Discussion by TomWright** [FirstGearGames/FishNet#210] [Feature Request]
Add options to `SceneLoadData` to allow the server to change the active scene on server, client or both when loading scenes over the network.

Context: https://discord.com/channels/424284635074134018/1034477094731784302/1060977150519423006

**Comment by Katerlad** [comment]
Whoop Whoop! Full customization of which scene is the active scene for client/server sounds great!!

**Comment by FirstGearGames** [comment]
Added in 3.1.1

**Reply by Katerlad** [reply]
I mentioned this in your stream, but I believe this suggestion was misinterpreted. This was a separate Suggestion on top of Preferred Active Scene to be able to customize if you want the server to set active scene only, the client to, or both.. @TomWright correct me if I am wrong but this feature was not added.

**Reply by Katerlad** [reply]
![image](https://user-images.githubusercontent.com/54480322/216681018-634728c5-ed3c-4aa7-9560-ed363874ab9f.png)

**Reply by TomWright** [reply]
I haven't actually seen the release yet but will take a look when I get a chance. Work has been busy so I haven't spent any time recently looking at FN.
